pkg/apis/serving/k8s_validation_test.go

```diff
@@ -1198,6 +1199,39 @@ func TestVolumeValidation(t *testing.T) {
 			apis.ErrMissingField("projected[0].secret.items[0].path")).Also(
 			apis.ErrMissingField("projected[1].configMap.items[0].key")).Also(
 			apis.ErrMissingField("projected[1].configMap.items[0].path")),
+	}, {
+		name: "serviceaccounttoken",
+		v: corev1.Volume{
+			Name: "foo",
+			VolumeSource: corev1.VolumeSource{
+				Projected: &corev1.ProjectedVolumeSource{
+					Sources: []corev1.VolumeProjection{{
+						ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+							Audience:          "api",
+							ExpirationSeconds: &expirationSeconds,
+							Path:              "token",
+						},
+					},
+					}},
```

Comment:
nit:

```suggestion
					}},
				},
```

Same below.